.github/workflows/ci.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest]
-        node-version: [12.x]
```

Comment:
Do any of the docker images need to be updated? Upon a quick glance it looks like the [common ts dockerfile](https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/build/docker/ts/Dockerfile#L1) uses `node:lts-alpine`, which actually points to node.js 16 currently. We might want to pin that to a fully qualified tag. As far as I can tell the other TS dockerfiles are pinned to `node:14.17-alpine3.13`, so I think those are fine

Reply:
Updated. PTAL